library_generation/main.py

```diff
@@ -0,0 +1,65 @@
+"""
+parses a config yaml and generates libraries via generate_composed_library.py
+"""
+
+import click
+from generate_composed_library import generate_composed_library
+from typing import Dict
+from model.GenerationConfig import GenerationConfig
+from model.Library import Library
+from model.GAPIC import GAPIC
+from collections.abc import Sequence
+from absl import app
+
+@click.group(invoke_without_command=False)
+@click.pass_context
+@click.version_option(message="%(version)s")
+def main(ctx):
+    pass
+
+@main.command()
+@click.option(
+    "--generation-config-yaml",
+    required=True,
+    type=str,
+    help="""
+    Path to generation_config.yaml that contains the metadata about library generation
+    """
+)
+@click.option(
+    "--repository-location",
+    required=False,
+    type=str,
+    help="""
+    Path to repository where generated files will be merged into, via owlbot copy-code.
+    Specifying this option enables postprocessing
```

Comment:
What happens if I set `repository-location` and set `enable-postprocessing` to `False`?

Reply:
I removed this option (commit upcoming)

Reply:
I modified the behavior. We will always have a repository downloaded, either passed via `--repository-path` or automatically downloaded if not specified. `enable_postprocessing`  will be safe in case `--repository-path` is not specified/set-to-false.